cms/djangoapps/modulestore_migrator/api.py

```diff
@@ -0,0 +1,57 @@
+"""
+API for migration from modulestore to learning core
+"""
+from opaque_keys.edx.locator import LibraryLocatorV2
+from opaque_keys.edx.keys import LearningContextKey
+from openedx_learning.api.authoring import get_collection
+
+from openedx.core.djangoapps.content_libraries.api import get_library
+from openedx.core.types.user import AuthUser
+
+from . import tasks
+from .data import RepeatHandlingStrategy
+from .models import ModulestoreSource
+
+
+__all__ = (
+    "start_migration_to_library",
+)
+
+
+def start_migration_to_library(
+    *,
+    user: AuthUser,
+    source_key: LearningContextKey,
+    target_library_key: LibraryLocatorV2,
+    target_collection_slug: str | None = None,
+    composition_level: str,
+    repeat_handling_strategy: str,
+    preserve_url_slugs: bool,
+    forward_source_to_target: bool,
+) -> None:
```

Comment:
Nit: we are returning `AsyncResult` from `celery.result` here
```suggestion
) -> AsyncResult:
```

Reply:
@rpenido Updated. Thanks!